### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by khoingt** [hxr404/Discord-Console-hacks#11] [Q&A] [answered]
Hi, I just stumble upon this and, tbh, it's great!
I'm just wondering that, can I get individual badges?
Thanks :)

**Comment by hxr404** [comment, marked as answer]
sorry, I just noticed that I didn't reply....

Yes, you can get individual badges. You just have to replace the `-1` with the corresponding number

**Comment by RobinRMC** [comment]
This thread should be marked as Answered.
